packages/plugins/model-define/bayesian-model-define/src/inference.ts

```diff
@@ -0,0 +1,22 @@
+import { CsvSample } from '@pipcook/pipcook-core';
+import { join } from 'path';
+import { processPredictData, loadModel } from './script';
+
+const boa = require('@pipcook/boa');
+const sys = boa.import('sys');
+
+/**
+ * The inference model.
+ * @param recoverPath The model path to be recovered.
+ */
+export default async function inference (recoverPath: string): Promise<any> {
```

Comment:
Shall we give a type definition for inference function in pipcook-core

Reply:
Of course :)